### DISCUSSION TITLE
Upload CSV - Feathers 5 with Koa

### DISCUSSION THREAD

**Discussion by arcanys-ivan** [feathersjs/feathers#3097] [Q&A]
How to upload a file in the latest feathers 5 with Koa transport. It seems like I cant get the files from the request, inside middleware, hooks or the service itself. Im not sure if Multipart Form is still enabled, but bodyParser has no option for that now.

Is this something to be implemented yet?
I would like to import a CSV file to a service. And i am stuck with this one.

Thanks

**Comment by 8BallBomBom** [comment]
Once the next update goes out for `Feathers v5` the Koa body parsing middleware should be updated to another package which is much newer and supports multipart form data.
Have a look over [here](https://github.com/feathersjs/feathers/pull/3093).

As for the guide?
That does indeed need updating to explain how to go about implementing uploads with Koa.
But also `feathers-blob` appears to need an update to support the new feathers.

**Reply by arcanys-ivan** [reply]
Thanks for your thoughts, and I found a way myself. 
Just checked the `Busboy` library, whic multer is using it as well, but you had to create the implementation instead, (multer did it). 
:) 

But yes, might be worth if the feathers documented it if its ready, or should at least have a tutorial on it.

**Reply by 8BallBomBom** [reply]
Pretty much a case of apples and oranges, whatever gets the job done 👍🏻 but once the latest update goes out you'll be able to use the `koa-body` package imported as `bodyParser` from `@feathersjs/koa` which will essentially give you the same functionality.

**Reply by arcanys-ivan** [reply]
still figuring it out how to make it work with just the bare Busyboy library, seems like theres some weird implementation in the middleware,

**Reply by marcintosh87** [reply]
Any luck with this? I am running into the same issue.

**Comment by SYBIOTE** [comment]
any updates on this? want to upload a .glb file using featherjs/koa
